docs/layout-of-source-files.rst

```diff
@@ -338,24 +338,3 @@ Additionally, there is another type of comment called a NatSpec comment,
 which is detailed in the :ref:`style guide<natspec>` section. They are written with a
 triple slash (``///``) or a double asterisk block(``/** ... */``) and
 they should be used directly above function declarations or statements.
-
-In the following example we document the title of the contract, the explanation
```

Comment:
Removed this as the `layout-of-source-files` does not go into detail in other sections, and this section links to the style guide which has an example already. Lastly it is one less example to maintain.